jdk/test/java/lang/ProcessBuilder/Basic.java

```diff
@@ -1663,6 +1669,10 @@ public void doIt(Map<String,String> environ) {
             List<String> list = new ArrayList<String>(javaChildArgs);
             list.add(1, String.format("-XX:OnOutOfMemoryError=%s -version",
                                       javaExe));
+            // Disable OpenJ9 OOM dumps for this OOM test, but enable others to catch unexpected problems
+            list.add(1, "-Xdump:system:events=gpf+abort+traceassert+corruptcache");
+            list.add(1, "-Xdump:system:none");
+            list.add(1, "-Xdump:heap:none");
```

Comment:
This might have been easier to follow if the options were added in order:
```
	list.add(2, "-Xdump:heap:none");
	list.add(3, "-Xdump:system:none");
	list.add(4, "-Xdump:system:events=gpf+abort+traceassert+corruptcache");
```
(and the comment should end with a period).

Reply:
Updated and re-tested. The other versions will follow.